framework/src/test/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStoreTest.java

```diff
@@ -29,13 +30,15 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.ResourceLock;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.test.util.ReflectionTestUtils;
 import uk.gov.ons.ctp.common.error.CTPException;
 
 @ExtendWith(MockitoExtension.class)
+@ResourceLock(value = "FirestoreDataStoreTest", mode = READ_WRITE)
 public class FirestoreDataStoreTest extends CloudTestBase {
 
   private static FirestoreDataStore firestoreDataStore = new FirestoreDataStore();
```

Comment:
Just remove static here, then we don't need to put a resource lock on

Reply:
Agree. Changing